doc/developer_information.rst

```diff
@@ -59,6 +59,12 @@ In the following example, we'll use the file ``restic-0.14.0.tar.gz`` and Go
     $ touch --reference VERSION restic_0.14.0_windows_amd64.exe
     $ TZ=Europe/Berlin zip -q -X restic_0.14.0_windows_amd64.zip restic_0.14.0_windows_amd64.exe
 
+4. To disable ``self-update`` functionality, e.g. for distribution package builds where package contents are not to be modified by users, disable the ``selfupdate`` tag:
```

Comment:
The `selfupdate` tag is not set by default. Thus `selfupdate` is already disabled by default.

Reply:
Ah yes, I confused myself when digging into this.
This is entirely redundant then and I'll omit this commit.